src/wrappers.jl

```diff
@@ -33,13 +35,23 @@ CommonRLInterface.actions(w::MyActionWrapper) = [-1, 1]
 actions(w) # will now return [-1, 1]
 ```
 """
-abstract type AbstractWrapper end
+abstract type AbstractWrapper <: AbstractEnv end
 
 """
-Indicate what the wrapped environment is for an AbstractWrapper (see the AbstractWrapper docstring)
+Indicate what the wrapped environment is for an AbstractWrapper (see the AbstractWrapper docstring).
 """
 function wrapped_env end
 
+"""
+Returns the recursively unwrapped environment.
+"""
+function base_env(env::AbstractWrapper)
+    while env isa AbstractWrapper
+        env = wrapped_env(env)
+    end
+    return env
+end
+
```

Comment:
Should we additionally define
```
base_env(env::AbstractEnv) = env
```
So there won't be an error on envs that are not wrappers?